src/components/backend-ai-storage-proxy-list.ts

```diff
@@ -52,6 +54,7 @@ export default class BackendAIStorageProxyList extends BackendAIPage {
   @property({type: Object}) _boundCapabilitiesRenderer = this.capabilitiesRenderer.bind(this);
   @property({type: Object}) _boundControlRenderer = this.controlRenderer.bind(this);
   @property({type: String}) filter = '';
+  @property({type: Number}) _totalAgentCount = 0;
```

Comment:
This name should be changed since _totalAgentCount property is already in use at `backend-ai-agent-list`.